faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -919,6 +919,26 @@ class ClientSpec
       identifyR.to[Boolean].get shouldBe true
       }
 
+  it should "test CurrentToken with internal token" in {
+    val createR = client.query(Create(Collection("spells"), Obj("credentials" -> Obj("password" -> "abcdefg")))).futureValue
```

Comment:
Also, I forgot about this in the first pass, but foreseeing some upcoming changes (specifically the ones in ticket [DRV-152](https://faunadb.atlassian.net/browse/DRV-152)), I think it would be better if we keep all of the test specific fixtures within the test case.

In other words, instead of reusing the `"spells"` Collection, I'd say we create a new Collection with some random name here. The `"spells"` Collection is actually not used by the majotiery of the test cases, so I wouldn't be counting with it ending up in a common shared fixture setup in the future! :)